SourceryTests/Parsing/FileParserSpec.swift

```diff
@@ -98,17 +99,6 @@ class FileParserSpec: QuickSpec {
                             ]))
                         }
 
-                        it("extracts method with closure parameter") {
```

Comment:
There was already one for that case, I also extended it with dictionary literal and generic type